docs/upgrade-guides/3-16-to-3-17.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: Upgrading from 3.16 to 3.17
+sidebar_position: 8
+---
+
+:::info
+To follow the zero-downtime strategy when upgrading to 3.17, **It is recommended to first migrate to at least 3.16.2** and turn on the Celery worker to process all data migrations asynchronously.
```

Comment:
@IKarbowiak I think you should link to the [Zero downtime migrations](https://docs.saleor.io/docs/3.x/developer/community/zero-downtime-migrations)

Reply:
IMO we shouldn't as this page is for developers about writing zero downtime migrations, it seems useless for someone that want to do the upgrade.

Reply:
Maybe we are missing page describing what zero-downtime strategy is

Reply:
https://github.com/saleor/saleor-docs/issues/965